go/README.md

```diff
@@ -29,16 +29,35 @@ Dremio Client Example.
 
 Usage:
   example -h | --help
-  example [--host=<hostname>] [--port=<port>] --user=<username> --pass=<password>
-          [--query=<query>] [--tls] [--certs=<path>]
+  example [--host=<hostname>] [--port=<port>] (--user=<username> --pass=<password> | --pat=<pat>)
+          [--tls] [--certs=<path>] [--query <query>] [--project_id=<project_id>]
 
 Options:
   -h --help           Show this help.
   --host=<hostname>   Dremio coordinator hostname [default: localhost]
   --port=<port>       Dremio flight server port [default: 32010]
   --user=<username>   Dremio username
   --pass=<password>   Dremio password
-  --query=<query>     SQL Query to test.
+  --pat=<pat>         Dremio personal access token
+  --query <query>     SQL Query to test.
   --tls               Enable encrypted connection.
   --certs=<path>      Path to trusted certificates for encrypted connection.
+  --project_id=<project_id>   Dremio project ID
 ```
+
+## Example
+
+If you a local instance of Dremio running on `localhost:32010` with a user `dremio` and password `dremio123`, 
```

Comment:
If you run a local instance of Dremio on localhost ?

Reply:
Had a typo good catch 👍